x-pack/plugins/cases/public/containers/api.ts

```diff
@@ -478,20 +479,50 @@ export const deleteFileAttachments = async ({
   });
 };
 
+export type FeatureIdsResponse = estypes.SearchResponse<
+  unknown,
+  {
+    consumer: {
+      buckets: Array<{ key: string; doc_count: number }>;
+    };
+    producer: {
+      buckets: Array<{ key: string; doc_count: number }>;
+    };
+  }
+>;
+
 export const getFeatureIds = async ({
   query,
   signal,
 }: {
-  query: { registrationContext: string[] };
+  query: {
+    ids: {
+      values: string[];
+    };
+  };
   signal?: AbortSignal;
-}): Promise<ValidFeatureId[]> => {
-  return KibanaServices.get().http.fetch<ValidFeatureId[]>(
-    `${BASE_RAC_ALERTS_API_PATH}/_feature_ids`,
-    {
-      signal,
+}): Promise<FeatureIdsResponse> => {
+  return KibanaServices.get().http.post<FeatureIdsResponse>(`${BASE_RAC_ALERTS_API_PATH}/find`, {
```

Comment:
I think is better to create a new endpoint. Our internal endpoints are not versioned and we have to version this endpoint to be able to do a breaking change. Also, internal APIs should be fully backward compatible. By introducing a new API we will bypass this issue. Wdyt?

Reply:
I won't do that, we will move them to versioning at one point.

Reply:
We talked offline and I was confused. I thought the code was on the server side and we were changing the response of the backend. All good. Sorry for the confusion.